csrankings-r.csv

```diff
@@ -892,6 +892,7 @@ Robert P. Burton,Brigham Young University,https://cs.byu.edu/faculty/rpburton,0N
 Robert P. Dick,University of Michigan,https://ece.engin.umich.edu/personnel/dick-robert,p2I_CecAAAAJ
 Robert Pastel,Michigan Technological University,https://www.mtu.edu/cs/department/faculty-staff/faculty/pastel,NOSCHOLARPAGE
 Robert Patro,University of Maryland - College Park,https://www.cs.umd.edu/people/nomad,H36hOqEAAAAJ
+Robert Piché,Tampere University,https://www.tuni.fi/en/robert-piche,Z8UrQ0EAAAAJ

```

Comment:
Teaching faculty and visitors are not eligible for inclusion.

Reply:
Indeed -- we'll remove this one.

Thank you for reviewing the PR! We'll get these changes in next week.

Reply:
Addressed via commit 136e8e1aa2